package.json

```diff
@@ -3,18 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
-    "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/react-fontawesome": "^0.1.16",
-    "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/user-event": "^13.5.0",
+    "@popperjs/core": "^2.10.2",
     "bootstrap": "^5.1.3",
-    "react": "^17.0.2",
+    "react": "17.0.2",
     "react-bootstrap": "^2.1.1",
-    "react-dom": "^17.0.2",
-    "react-scripts": "5.0.0",
```

Comment:
`react-scripts` is a package that includes scripts and configuration for `create-react-app`. Removing it will cause issues with npm scripts like `start`, `build`, `test` and break the app.

Reply:
Okay??
This is right.
What is the issue here?

Reply:
> Okay?? This is right. What is the issue here?

Well, they're removed.

Reply:
I see my mistake. I will put them back.

Reply:
I have added back the dependencies I had removed.